h2o-algos/src/main/java/hex/glm/ComputationState.java

```diff
@@ -530,10 +647,27 @@ public boolean converged(){
   }
 
   protected double updateState(double [] beta,GLMGradientInfo ginfo){
-    _betaDiff = ArrayUtils.linfnorm(_beta == null?beta:ArrayUtils.subtract(_beta,beta),false);
+    _betaDiff = _activeColsAll==null
```

Comment:
please replace multi line ternary operator with if/else, also looks like a candidate for extract to method